src/Jackalope/Transport/DoctrineDBAL/CachedClient.php

```diff
@@ -147,10 +205,14 @@ public function getAccessibleWorkspaceNames()
      */
     public function getNode($path)
     {
+        if (empty($this->caches['nodes'])) {
+            return parent::getNode($path);
```

Comment:
also this change is actually part of #280 

to review this maybe better to look at https://github.com/jackalope/jackalope-doctrine-dbal/compare/cache_queries...cache_queries_node_types